### DISCUSSION TITLE
Thank you for this wonderful tool

### DISCUSSION THREAD

**Discussion by ramzahn** [kangyu-california/PersistentWindows#139] [General]
I just want to unload my heap of gratitude. 

Especially in our changing times, working with three monitors at work, with one monitor(laptop) when using puplic transportation, and with 2 monitors at home office, it never lets me down.

Let me state it again. This is an excellent tool.

Strangely enough it seems to be the only one of its kind. I cannot believe that nobody felt the urge to build such a tool. Obviously nobody took it unto himself. 

You did!

Thank you.

**Comment by kangyu-california** [comment]
Thanks for the feedback, just made a new release 5.30, happy holidays!
